src/components/ebay-video/component.ts

```diff
@@ -58,6 +64,10 @@ interface VideoInput extends Omit<Marko.HTML.Video, `on${string}`> {
     clip?: any[];
     source: Marko.AttrTag<Marko.HTML.Source>;
     "report-text"?: AttrString;
```

Comment:
Can you include an alias for this?
```suggestion
    /** @deprecated use `a11y-report-text` instead */
    "report-text"?: AttrString;
    "a11y-report-text"?: AttrString;
```